packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -2232,53 +2159,73 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin,
     _setSelection(nextSelection, cause);
   }
 
-  // Handles shortcut functionality including cut, copy, paste and select all
-  // using control/command + (X, C, V, A).
-  Future<void> _handleShortcuts(LogicalKeyboardKey key) async {
+  /// Set the current [selection] to contain the entire text value.
+  ///
+  /// {@macro flutter.rendering.RenderEditable.cause}
+  void selectAll(SelectionChangedCause cause) {
+    _setSelection(
+      selection!.copyWith(
+        baseOffset: 0,
+        extentOffset: textSelectionDelegate.textEditingValue.text.length,
+      ),
+      cause,
+    );
+  }
+
+  /// Copy current [selection] to Clipboard.
```

Comment:
Nit: Here and everywhere else it appears in docs, I think Clipboard should be linked (`[Clipboard]`).